main.js

```diff
@@ -26,10 +26,16 @@ exports.active = function(item) {
 };
 
 // That's not how node.js implements it but the exposed api is the same.
-exports.setImmediate = function(fn) {
-  var id = nextImmediateId++;
-  var args = arguments.length < 2 ? false : slice.call(arguments, 1);
+exports.setImmediate =function(fn) {
```

Comment:
Is it sufficient to do:
```js
exports.setImmediate = setImmediate || function(fn) { ...
```
instead? This way you don't add an extra function when it does exist.